digdag-standards/src/test/resources/io/digdag/standards/operator/for_each/basic_expected.yml

```diff
@@ -1,6 +1,6 @@
-+for-a=1:
++for-0=a=0=1:
```

Comment:
The first `0` is index of the key. The second `0` is the index of the value. Both together unambiguously identify a key/value pair.